src/explore-education-statistics-admin/src/pages/publication/components/PublicationManageReleaseTeamAccess.tsx

```diff
@@ -0,0 +1,151 @@
+import ButtonLink from '@admin/components/ButtonLink';
+import UserReleaseRoleTable from '@admin/pages/publication/components/ReleaseUserTable';
+import { getReleaseApprovalStatusLabel } from '@admin/pages/release/utils/releaseSummaryUtil';
+import {
+  PublicationTeamRouteParams,
+  publicationManageReleaseContributorsPageRoute,
+} from '@admin/routes/publicationRoutes';
+import { ReleaseSummary } from '@admin/services/releaseService';
+import releasePermissionService, {
+  UserReleaseRole,
+  UserReleaseInvite,
+} from '@admin/services/releasePermissionService';
+import userService from '@admin/services/userService';
+import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import Tag from '@common/components/Tag';
+import WarningMessage from '@common/components/WarningMessage';
+import React from 'react';
+import { generatePath } from 'react-router-dom';
+import SummaryList from '@common/components/SummaryList';
+import SummaryListItem from '@common/components/SummaryListItem';
+
+interface Props {
+  publicationId: string;
+  release: ReleaseSummary;
+  showManageContributorsButton: boolean;
+}
+
+const PublicationManageReleaseTeamAccess = ({
+  publicationId,
+  release,
+  showManageContributorsButton,
+}: Props) => {
+  const {
+    value,
+    isLoading,
+    setState: setUsersAndInvites,
+  } = useAsyncHandledRetry<{
```

Comment:
Nice.  I've applied it to this particular component (now named `PublicationReleaseAccess.tsx`) and its tests.  Note that it also had a knock-on effect onto the `PublicationTeamAccessPage.test.tsx` tests too, in that we're now using `waitfor` to check for rendered DOM elements within the child `PublicationReleaseAccess` component rather than on elements within the `PublicationTeamAccessPage.tsx` itself - this is to give the child component enough time to render properly before further checking its content in the test.  I guess switching `useAsyncHandlerRetry` out for `useQuery` added a tiny bit extra render time to the component, whilst running within the Jest harness at least.

Reply:
I'm not going to roll this out to any other components as part of this PR, as I really need to get this one out of the door, but I've made folks aware of this new direction in standup this morning.